tests/test_import_status_endpoint.py

```diff
@@ -168,10 +175,7 @@ def should_report_s3_batch_upload_failures(
     }
 
     describe_s3_job_mock.return_value = {
-        "Job": {
-            "Status": "Completed",
-            "FailureReasons": [{"FailureCode": "TEST_CODE", "FailureReason": "TEST_REASON"}],
-        }
+        "Job": {"Status": S3_BATCH_STATUS_COMPLETE, "ProgressSummary": {"NumberOfTasksFailed": 1}}
```

Comment:
Shouldn't we still keep the original test with `FailureReasons`? I feel like `NumberOfTasksFailed` warrants a separate but very similar test.